### DISCUSSION TITLE
PFM half16 variant

### DISCUSSION THREAD

**Discussion by walbourn** [ImageMagick/ImageMagick#2108] [Development]
[This post](https://oyranos.org/2015/03/portable-float-map-with-16-bit-half/index.html) suggests there should be a half16 variant of the PFM format. It's pretty simple to implement, but when I tried this with imagemagick, it still wrote it as a float32 file:

```
magick convert memorial.hdr -depth 16 memorial_half.pfm
```

**Comment by snibgo** [comment]
That post, written in 2015, was a "request for comment" for a standard 16-bit PFM variant. Apparently, there was no standard for that. Yes, it would be simple. But somewhat pointless if no other software could read/write that variant.

**Reply by urban-warrior** [reply]
Agreed.  Unless there is a formal specification for the variant we would not be interested in supporting 16-bit PFM.  Doing so would just break other PFM readers.

**Comment by urban-warrior** [comment]
Check the latest Github master.  It includes support for 16-bit half float.  Try

`magick convert memorial.hdr memorial_half.phm`

We have not tried it against other packages that support this format.  Fingers-crossed that it works :-).
